### PR TITLE
fix: fixed css and added responsive design

### PR DESCRIPTION
**Pull request by Shubhamkr2610**
added mobile responsive design 
image

image

### CODE REVIEW

File: src/components/header/header.css [hunk 1]
```diff
@@ -1,4 +1,5 @@
 .header {
+    width: 100%;
     color: #F40075;
     border-bottom: 2px solid #F40075 ;
     background-color: white;
```
Comment (arjun-computer-geek) [inline src/components/header/header.css:5]: use color variables

**Review by arjun-computer-geek — APPROVED** [review]
find nothing much. Good code of conduct. cleaner code structure.

File: src/components/header/header.css [hunk 2]
```diff
@@ -52,10 +54,13 @@
...
+        width: 100%;
+    }
 
     .brand-title {
         font-size: 10vw;
-        /* margin-left: 1rem; */
+        margin-left: 5px;
     }
   
     .brand-title img{
```
Comment (Rockyraven) [inline src/components/header/header.css:66]: remove unwanted code

**Review by Rockyraven — APPROVED** [review]
your code is clean and easy to readable 
You can good to go

**Pull request merged**